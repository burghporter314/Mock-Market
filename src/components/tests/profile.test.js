import React from 'react';
import Profile from '../Profile';
import renderer from 'react-test-renderer';

it('should match snapshot', () => {
    const component = renderer.create(
        <Profile/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})