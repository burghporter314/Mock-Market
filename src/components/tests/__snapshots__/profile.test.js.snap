exports[`test should match snapshot 1`] = `null`;
